docs/src/api.md

```diff
@@ -3,15 +3,12 @@ CurrentModule = DifferentiationInterface
 CollapsedDocStrings = true
 ```
 
-# Interface
+# API reference
 
 ```@docs
 DifferentiationInterface
 ```
 
```

Comment:
It might be nice to show the list of backends ADTypes provides. Do we do this? 
It should be possible by including ADTypes in the `modules` argument of `makedocs`.

Reply:
Yeah but we don't want to show everything from ADTypes, only the ones we use. I'll add those

Reply:
Solved in #31